### PR TITLE
Fixes proc statements followed by comments & indented multiline comments

### LINKED ISSUE #30: Possible regression with multiline comments
```
    if (cellcontrol.var_swap)
			var/airtemp = 0
			/*
			for(var/turf/T in src.FindLinkedTurfs())
				if (istype(T, /turf/space))
					space = 1
					src.airforce = src.oxygen + src.poison + src.n2 + src.co2 + 25000
					src.airdir = get_dir(src, T)
					airtemp = src.temp
					break // *****RM
				else
					divideby++
					total += T.oldoxy
					tpoison += T.oldpoison
					tco2 += T.oldco2
					tosl_gas += T.osl_gas
					ton2 += T.on2
					totemp += T.otemp
					if (T.firelevel >= config.min_gas_for_fire)
						burn = 1
					adiff = src.oldoxy + src.oldco2 + src.on2 - (T.oldoxy + T.oldco2 + T.on2)
					if (adiff > src.airforce)
						src.airforce = adiff
						src.airdir = get_dir(src, T)
						airtemp = src.otemp

			*/
			if(airN)
				if(istype(linkN, /turf/space) && (!config.air_pressure_flow))
					space = 1
					src.airforce = src.oxygen + src.poison + src.n2 + src.co2 + 25000
					src.airdir = NORTH
					airtemp = src.temp
```
Currently the line `if(airN)` throws an "Expected dedent" error. Multiline comments in the middle of indented code blocks seem to screw with indent parsing, probably caused by #25

### PR DESCRIPTION
**Pull request by ike709**
Currently, if you have a comment after a proc statement it will throw.

Ex: 
```
for(var/A in thing)
    if(A + 2 > 5)
        continue //Here is a comment. This would cause an exception.
    return thing[A]
```

Also fixes #30

### CODE REVIEW

**Review by wixoaGit — CHANGES_REQUESTED** [review]

File: OpenDreamShared/Compiler/DM/DMParser.cs [hunk 1]
```diff
@@ -357,6 +357,9 @@ class DMParser : Parser<Token> {
                 if (procStatement == null) procStatement = DoWhile();
                 if (procStatement == null) procStatement = Switch();
 
+                Whitespace();
+                Check(TokenType.Skip);
```
[the remark this answers is not in the document]
Comment (ike709) [reply OpenDreamShared/Compiler/DM/DMParser.cs:361]: What's the ideal way to ignore a comment, then?

[the remark this answers is not in the document]
Comment (wixoaGit) [reply OpenDreamShared/Compiler/DM/DMParser.cs:361]: You don't need to do anything, the comment is already ignored.

[the remark this answers is not in the document]
Comment (ike709) [reply OpenDreamShared/Compiler/DM/DMParser.cs]: Ah, it's just the trailing whitespace.

**Comment by ike709** [comment]
Should be good to go.

**Review by wixoaGit — APPROVED** [review]

**Pull request merged**